x/auth/ante/sigverify.go

```diff
@@ -447,3 +472,40 @@ func CountSubKeys(pub cryptotypes.PubKey) int {
 
 	return numKeys
 }
+
+// signatureDataToBz converts a SignatureData into raw bytes signature. It is
+// the same function as in auth/tx/sigs.go, but copied here because of import
```

Comment:
Is there a way we can avoid copying? Could we move the original code elsewhere?

Reply:
I didn't come up with a solution that's not API-breaking.

Move the original code might work, or I also proposed https://github.com/cosmos/cosmos-sdk/pull/9750#discussion_r674794051.